### PR TITLE
Remove latin script default from @bbc-psammead/timestamp

### LINKED ISSUE #481: Remove latin script default from `@bbc/psammead-timestamp`
**Is your feature request related to a problem? Please describe.**
We should make some things required to avoid bloating the bundle

**Describe the solution you'd like**
Remove the latin import and make script a required value. We can still keep the `getBrevier` default, as passing in a script is unusual compared to other packages.

**Testing notes**
[Tester to complete]

Dev insight: Regression in storybook

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #481 

**Overall change:** _Removes latin script default from `@bbc-psammead/timestamp`._

**Code changes:**

- _Remove latin import._
- _Make script a required prop._
- _Update test by adding script prop._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by DenisHdz — DISMISSED** [review]

**Comment by DenisHdz** [comment]
Just seen it is still in progress, but it looks fine :)

File: packages/components/psammead-timestamp/CHANGELOG.md [hunk 1]
```diff
@@ -3,5 +3,6 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.3.0 | [PR#494](https://github.com/bbc/psammead/pull/494) Remove script import. |
```
Comment (ghost) [inline packages/components/psammead-timestamp/CHANGELOG.md]:
```suggestion
| 0.3.0 | [PR#494](https://github.com/bbc/psammead/pull/494) Make script a required prop. |
```

**Review by ghost — COMMENTED** [review]
Just one minor change.

**Review by ghost — APPROVED** [review]
some

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Comment by sareh** [comment]
@Bopchy @pjlee11 @DenisBBC  @sadickisaac Hi everyone - this PR didn't increment the `package.json` version (nor the `package-lock.json` version). As a result, it was not published by CI to npm. https://www.npmjs.com/package/@bbc/psammead-timestamp

Can we please in future take note of this. Can someone please open up a PR to increment the version number? Thank you.

**Pull request merged**